proto/ibc/client/client.proto

```diff
@@ -21,3 +21,20 @@ message ClientConsensusStates {
   // consensus states associated with the client
   repeated google.protobuf.Any consensus_states = 2 [(gogoproto.moretags) = "yaml:\"consensus_states\""];
 }
+
+// Height is a monotonically increasing data type
+// that can be compared against another Height for the purposes of updating and freezing clients
+//
+// Normally the EpochHeight is incremented at each height while keeping epoch number the same
+// However some consensus algorithms may choose to reset the height in certain conditions
+// e.g. hard forks, state-machine breaking changes
+// In these cases, the epoch number is incremented so that height continues to be monitonically increasing
+// even as the EpochHeight gets reset
+message Height {
+  option (gogoproto.goproto_stringer) = false;
```

Comment:
why this option?

Reply:
I want to be able to define my own `String()` method on Height, especially since it will eventually go into `host.ConsensusStatePath`